benchmark/buffers/buffer-concat.js

```diff
@@ -2,20 +2,20 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  pieces: [1, 4, 16],
+  pieces: [4, 16],
   pieceSize: [1, 16, 256],
   withTotalLength: [0, 1],
-  n: [1024]
+  n: [8e5]
 });
 
 function main({ n, pieces, pieceSize, withTotalLength }) {
-  const list = new Array(pieces);
-  list.fill(Buffer.allocUnsafe(pieceSize));
+  const list = Array.from({ length: pieces })
```

Comment:
Using `Array.from` results in a different internal `map` type. It's a dense array instead of a sparse one.